### PR TITLE
Add new recovery measure

### PR DESCRIPTION
**Pull request by mkangia**
https://app.asana.com/0/898060497005556/956316009861912

### CODE REVIEW

**Comment by mkangia** [comment]
adding hold of on merge to test this on staging with shubham once.

**Review by millerdev — APPROVED** [review]

**Comment by shubham1g5** [comment]
@mkangia tested this. Working great 👍

**Comment by mkangia** [comment]
Thank you @shubham1g5

File: corehq/apps/ota/migrations/0008_auto_20190108_1808.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('ota', '0007_update_blob_paths'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='mobilerecoverymeasure',
+            name='measure',
+            field=models.CharField(choices=[('app_reinstall_and_update', 'Reinstall and Update App'), ('app_update', 'Update App'), ('cc_reinstall', 'CC Reinstall Needed'), ('cc_update', 'CC Update Needed'), ('app_offline_reinstall_and_update', 'Offline Reinstall and Update App')], help_text='<strong>Reinstall and Update App:</strong> Reinstall the current CommCare app either OTA or with a ccz, but requiring an OTA update to the latest version before it may be used.<br/><strong>Update App:</strong> Update the current CommCare app<br/><strong>CC Reinstall Needed:</strong> Notify the user that CommCare needs to be reinstalled<br/><strong>CC Update Needed:</strong> Notify the user that CommCare needs to be updated<br/><strong>Offline Reinstall and Update App:</strong> Reinstall the current CommCare app offline.', max_length=255),
```
[the remark this answers is not in the document]
Comment (mkangia) [reply corehq/apps/ota/migrations/0008_auto_20190108_1808.py]: Ignoring this.

File: corehq/apps/ota/migrations/0008_auto_20190108_1808.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-01-08 18:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
```
[the remark this answers is not in the document]
Comment (mkangia) [reply corehq/apps/ota/migrations/0008_auto_20190108_1808.py]: resolved.

**Pull request merged**